tests/nosetests/pyanaconda_tests/module_services_test.py

```diff
@@ -23,14 +23,19 @@
 from textwrap import dedent
 from unittest.mock import patch
 
-from mock import Mock
+from mock import Mock, patch
```

Comment:
You should use the other `patch` imported above.

Reply:
Fixed.